isis/src/base/objs/SpiceRotation/unitTest.cpp

```diff
@@ -1,890 +1,350 @@
 #include <iostream>
 #include <iomanip>
-#include <cmath>
 
 #include <nlohmann/json.hpp>
 
 #include "FileName.h"
 #include "IException.h"
+#include "IString.h"
 #include "Preference.h"
-#include "SpiceRotation.h"
+#include "SpicePosition.h"
 #include "Table.h"
 
-// Declarations for bindings for Naif Spicelib routines that do not have
-// a wrapper
-extern int bodeul_(integer *body, doublereal *et, doublereal *ra,
-                   doublereal *dec, doublereal *w, doublereal *lamda);
-
 using json = nlohmann::json;
-using namespace std;
 using namespace Isis;
-
-//TODO test loadPCFromSpice() and loadPCFromTable() methods
-//TODO see end of unit test for exceptions that need to be tested
+using namespace std;
 
 int main(int argc, char *argv[]) {
   Preference::Preferences(true);
 
   cout << setprecision(8);
-  cout << "Unit test for SpiceRotation" << endl;
+  cout << "Unit test for SpicePosition" << endl;
```

Comment:
Why is all of this moved into the SpiceRotation unit test?

Reply:
Woah. I don't know how this happened, thanks for catching this. I'll fix it.